common/v2/components/WalletList/WalletList.tsx

```diff
@@ -105,7 +110,7 @@ export const WalletList = ({ wallets, onSelect, showHeader, calculateMargin }: P
               icon={walletInfo.icon}
               description={translateRaw(walletInfo.description)}
               margin={calculateMargin && calculateMargin(index)}
-              onClick={() => onSelect(wallet.name)}
+              onClick={() => onWalletClick(wallet.name)}
```

Comment:
Prefer naming the event handler by what it does instead of when https://medium.com/@vdsabev/simple-naming-tips-for-event-handlers-aeba8cf338ba
e.g `on` is reserved for the prop.

Reply:
Sound :+1: